docs/fastdds/logging/disable_module.rst

```diff
@@ -14,8 +14,7 @@ However, it is possible to fully disable each macro (and therefore each verbosit
 
 * |logInfo| is fully disabled by either:
 
-    * Setting CMake option ``CMAKE_BUILD_TYPE`` to something other than ``Debug`` (``Release`` or ``RelWithDebInfo``).
-    * Setting CMake option |LOG_NO_INFO| to ``ON``.
+    * Setting CMake option |LOG_NO_INFO| to ``ON`` (default with ``CMAKE_BUILD_TYPE`` other than ``Debug``).
```

Comment:
Write a period at the end of all the sentences in the lists

Reply:
:+1: